firebase/functions/index.js

```diff
@@ -124,6 +167,22 @@ exports.counter = functions.database.ref('/v2/results/{projectId}/{groupId}/{use
         })
     promises.push(contributions)
 
+    const contributionsNew = groupContributionsRefNew.once('value')
+        .then((dataSnapshot) => {
+            if (dataSnapshot.exists()) {
+                return null
+            }
+            else {
+            const data = {
+                'timestamp': result['timestamp'],
```

Comment:
looking at the app code, it seems that `timestamp` and `endTime` are the same value, maybe we should look into throwing `timestamp` away... unless you're modifying one of them in the backend functions?

Reply:
This is also a good comment and could lead us to a general discussion here. Do we actually need this information in firebase for each user?

Initially, we wanted to have this information to quanitify how much time a users spend on a specific group. But maybe it would be easier to put this information into the results generated by the app. E.g. by adding an attribute "timeSpent" instead of having "timestamp" and "endTime" which are the same.

And then we might even think about why we store the timestamps in firebase at all and not only the ids of the projects and groups a user has worked on. Having only this would also reduce data a lot and we currently do not use this information in firebase at all...